parsers/src/main/java/org/chocosolver/parser/RegParser.java

```diff
@@ -63,6 +63,13 @@ public abstract class RegParser implements IParser {
     )
     protected Level level = Level.COMPET;
 
+    @Option(name = "-log",
+            aliases = "--log-file",
+            usage = "Define the log file."
+    )
+    protected String logFile = "";
```

Comment:
I would suggest to rename it logFileName or logFilePath depending on what it is precisely

I would prefer to have null rather than "" to handle the case where there is none

Reply:
Done